apps/worldclock/world_clock.star

```diff
@@ -47,12 +47,7 @@ def main(config):
         ]
 
     location_count = int(config.get("location_count") or 3)
-
-    if location_count < 4 and len(locations) > 3:
-        locations.remove(locations[3])
```

Comment:
If we have two distinct locations A and B, and pass in location 1,2,3,4 = `A,B,A,A`, then this line will remove the first occurrence of location `A` (i.e. `location_1`) instead of the last one (`location_4`). Besides removing the wrong locations, this also breaks the assignment of the optional `location_x_label` fields.